docsrc/lsh.rst

```diff
@@ -123,3 +123,155 @@ bulk insertion.
 
 Note that querying the LSH object during an open insertion session may result in
 inconsistency.
+
+.. _minhash_lsh_async:
+
+Asynchronous MinHash LSH at scale
+---------------------------------
+
+.. note::
+    The module supports Python version >=3.6, and is currently experimental. 
+    So the interface may change slightly in the future.
+
+This module may be useful if you want to process millions of text documents 
+in streaming/batch mode using asynchronous RESTful API for clustering tasks, 
+and you expecting to maximize the throughput of your service.
+For example, this module allows your code to not get blocked by insertion. 
+
+We currently provide two implementations:
+* Asynchronous Redis storage (*python aioredis package*)
+* Asynchronous MongoDB storage (*python motor package*)
+
+Experimental result
+
+* Number of objects (insert, query): 12500
+* Pool: `concurrent.futures.ThreadPoolExecutor(max_workers=100)`
+* check_duplication: false
+* buffer_size: 500
+
+    +-------------------------+-------------------------------+------------------------+------------------------+
+    |                         | Synchronous tests, sec        | Asynchronous tests, sec                         |
+    |                         |                               +------------------------+------------------------+
+    |                         | *MinHashLSH*                  |*AsyncMinHashLSH*       || *ThreadPoolExecutor*  |
+    |                         | *redis storage*               |                        || *MinHashLSH*          |
+    |                         |                               |                        || *redis storage*       |
+    +=========================+===============================+========================+========================+
+    | **Insert**              |                               | 30.626                 | 41.023                 |
+    +-------------------------+-------------------------------+------------------------+------------------------+
+    || **Insert session**     | 6.729                         | 39.274                 |                        |
+    +-------------------------+-------------------------------+------------------------+------------------------+
+    | **Query**               | 112.155                       | 60.509                 | 68.283                 |
+    +-------------------------+-------------------------------+------------------------+------------------------+
+
+In summary, for faster querying use AsyncMinHashLSH module, 
```

Comment:
I fixed some English. Can you recompile this doc?

Reply:
Done. Additional, made some chenges inspired by yours